extension/package.json

```diff
@@ -27,6 +28,7 @@
     "react-virtualized": "^9.21.2",
     "remotedev": "^0.2.9",
     "typeface-inter": "^1.1.13",
+    "web-ext-types": "^3.2.1",
```

Comment:
We use `webextension-polyfill-ts` already for this purpose, so please remove the `web-ext-types` dependency unless you have found a particular case in which the former is not sufficient.

Reply:
removed